frontends/web/src/contexts/DarkmodeProvider.tsx

```diff
@@ -62,7 +70,13 @@ export const DarkModeProvider = ({ children }: TProps) => {
     setIsDarkMode(!isDarkMode);
```

Comment:
Not your changes, but I think it's fine to change it here. 

Could we change this bit to `setIsDarkMode(darkmode)` since it's already receiving the variable (`darkmode`). Just for consistency :)

Reply:
Done :+1: